### ISSUE TITLE
Top-Level Intermezzo

### ISSUE THREAD

**Issue by flexoron** [trealla-prolog/trealla#287]
v2.23.32
```
?- C = \ .
Error: syntax error, missing operand, user:1
   false.

?- C = (\) .
   C = (\).

?- C = \ .
Error: syntax error, missing operand, user:1
?-
% false. not printed

?- true.
   true.
?- C = \ .
Error: syntax error, missing operand, user:1
   true. % Now true?
?- 

?- false.
   false.
?- C = \ .
Error: syntax error, missing operand, user:1
   false. % and false again
?-

**Comment by infradig** [comment]
It shouldn't print true or false on an error.

**Comment by flexoron** [comment]
Fixed with v2.23.33
